### PR TITLE
Implement /advance command

### PR DESCRIPTION
**Pull request by WoozChucky**
Can be used on self or others, if no level is specified, it automatically advances the target's current level by 1.

### CODE REVIEW

File: src/Executables/Game/Commands/AdvanceCommand.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        }
+        else
+        {
+            if (context.Arguments.Level <= 0)
+            {
+                context.Arguments.Level = 1;
+            }
+            
+            // todo: cap level
+            target.SetPoint(EPoints.Level, (uint) (target.GetPoint(EPoints.Level) + context.Arguments.Level));
```
Comment (MeikelLP) [inline src/Executables/Game/Commands/AdvanceCommand.cs:37]: This does not check for max level. Maybe `SetPoint` should check it 🤔

**Review by MeikelLP — CHANGES_REQUESTED** [review]

File: src/Executables/Game/Commands/AdvanceCommand.cs (hunk 1 above)
Comment (WoozChucky) [reply src/Executables/Game/Commands/AdvanceCommand.cs:37]: Maybe could be a Game setting? Defaulting it to 120? 100? 99?

Comment (MeikelLP) [reply src/Executables/Game/Commands/AdvanceCommand.cs:37]: Yes. Right now the max level is determined by the experience manager afaik

File: src/Executables/Game/World/Entities/PlayerEntity.cs [hunk 2]
```diff
@@ -593,16 +608,15 @@ public override void SetPoint(EPoints point, uint value)
...
-                    Player.Level = (byte) value;
-                    foreach (var entity in NearbyEntities)
+                    var currentLevel = GetPoint(EPoints.Level);
+                    if (value > currentLevel)
                     {
-                        if (entity is IPlayerEntity other)
-                        {
-                            SendCharacterAdditional(other.Connection);
-                        }
+                        LevelUp((int) (value - currentLevel));
```
Comment (MeikelLP) [inline src/Executables/Game/World/Entities/PlayerEntity.cs:614]: Cant u just do `LevelUp(value - currentLevel)`? This covers increase and decrease without the if (?)

**Review by MeikelLP — CHANGES_REQUESTED** [review]

File: src/Executables/Game/World/Entities/PlayerEntity.cs (hunk 2 above)
Comment (WoozChucky) [reply src/Executables/Game/World/Entities/PlayerEntity.cs]: Yeah I don't know what I was thinking 🫨

File: src/Executables/Game/World/Entities/PlayerEntity.cs [hunk 3]
```diff
@@ -625,6 +616,7 @@ public override void SetPoint(EPoints point, uint value)
                 case EPoints.Gold:
                     Player.Gold = value;
                     break;
+                    break;
```
Comment (MeikelLP) [inline src/Executables/Game/World/Entities/PlayerEntity.cs:619]: Why 2x break?

**Review by MeikelLP — CHANGES_REQUESTED** [review]

**Review by MeikelLP — APPROVED** [review]

**Pull request merged**